examples/inference/benchmark_llama.py

```diff
@@ -136,9 +148,37 @@ def benchmark_inference(args):
     print_details_info(outputs, model.config, args, whole_end2end)
 
 
+def benchmark_auto_gptq_inference(args):
+    from auto_gptq import AutoGPTQForCausalLM
+
+    # load quantized model to the first GPU
+    model = AutoGPTQForCausalLM.from_quantized(args.gptq_model, device=torch.cuda.current_device())
```

Comment:
The only difference is model building process, right? I don't think it's essential to write two functions

Reply:
No, the generate arguments are also different.

Reply:
These all can be done by partial function

Reply:
Merge into one funtcion.